src/Aggregate/AggregateChanged.php

```diff
@@ -5,6 +5,7 @@
 namespace Patchlevel\EventSourcing\Aggregate;
 
 use DateTimeImmutable;
+use Patchlevel\EventSourcing\Clock;
```

Comment:
NIH? :slightly_smiling_face: https://github.com/lcobucci/clock

Reply:
We cannot do any service injection at this point. And features like sleep, so that you skip a certain period of time, are also not available.

Reply:
:disappointed: